picasso/src/test/java/com/squareup/picasso3/DispatcherTest.java

```diff
@@ -143,48 +143,23 @@
     assertThat(dispatcher.pausedActions).isEmpty();
 
     FetchAction fetchAction1 =
-        new FetchAction(mockPicasso(), new Request.Builder(URI_1).build(), pausedTag, null);
+        new FetchAction(mockPicasso(), new Request.Builder(URI_1).tag(pausedTag).build(), null);
     FetchAction fetchAction2 =
-        new FetchAction(mockPicasso(), new Request.Builder(URI_1).build(), pausedTag, null);
+        new FetchAction(mockPicasso(), new Request.Builder(URI_1).tag(pausedTag).build(), null);
     dispatcher.performSubmit(fetchAction1);
     dispatcher.performSubmit(fetchAction2);
 
     assertThat(dispatcher.pausedActions).hasSize(2);
   }
 
-  @Test public void performSubmitWithFetchActionWithSuccessCompletionCallback() {
-    String pausedTag = "pausedTag";
-    Callback callback = mockCallback();
-
-    FetchAction fetchAction =
-        new FetchAction(mockPicasso(), new Request.Builder(URI_1).build(), pausedTag, callback);
-    dispatcher.performSubmit(fetchAction);
-    fetchAction.complete(new RequestHandler.Result(bitmap1, MEMORY));
-
-    verify(callback).onSuccess();
-  }
-
-  @Test public void performSubmitWithFetchActionWithErrorCompletionCallback() {
-    String pausedTag = "pausedTag";
-    Callback callback = mockCallback();
-
-    FetchAction fetchAction =
-        new FetchAction(mockPicasso(), new Request.Builder(URI_1).build(), pausedTag, callback);
-    dispatcher.performSubmit(fetchAction, false);
-    Exception e = new RuntimeException();
-    fetchAction.error(e);
-
-    verify(callback).onError(e);
-  }
-
```

Comment:
These tests weren't useful, as they were checking whether fetchAction.complete/error called the appropriate callback method -- nothing to do with Dispatcher!